javascript/javascript-import-and-require.md

```diff
@@ -0,0 +1,79 @@
+# JavaScript - Require & Import
+
+## Project Time
+
+About 45 Minutes
+
+- 30 minutes for the video and reference material
+- 10 minutes for Check for Understanding
+- 5 minutes for verify
+
+## Prerequisites
+
+- JavaScript 7
+- [JavaScript Import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)

```

Comment:
Please remove Javascript `import` and `require` from the prerequisite. We cannot expect someone to already know something which we are going to explain.

Reply:
Yes, you can use this as a lesson material or additional Supplemental material instead.

Reply:
done